refactor: removed `Count` function with `ts_name` argument from src/tablet_client…

TAldridgeAO opened the pull request:
….h and src/tablet_client.cc. Removed ts_name parameter from count call in openmldb.cc

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This fixes Issue "Remove Count function from tablet client #1974"
close #1974 



* **What is the current behavior?** (You can also link to an open issue here)
No Behavior changes.


* **What is the new behavior (if this is a feature change)?**
No longer calls Count using ts_name in openmldb.cc

TAldridgeAO committed:
Removed Count function with 'ts_name' argument from src/tablet_client.h and src/tablet_client.cc. Removed ts_name parameter from count call in openmldb.cc

dl239 commented on src/client/tablet_client.cc:
```diff
@@ -779,28 +779,6 @@ bool TabletClient::Count(uint32_t tid, uint32_t pid, const std::string& pk, cons
     return Count(tid, pid, pk, idx_name, "", filter_expired_data, value, msg);
 }
 
-bool TabletClient::Count(uint32_t tid, uint32_t pid, const std::string& pk, const std::string& idx_name,
```
another `Count` function above will call this function. you can move this implement above.

TAldridgeAO replied:
Can you please clarify what you are wanting me to do with this? I can't find another function calling Count within this section.

TAldridgeAO replied:
Ignore that previous. I was looking at this pre-coffee. To clarify, it looks like the count function on line 777 should instead use the implementation from the removed Count function, correct?

dl239 replied:
yes,

TAldridgeAO replied:
Okay. I've just pushed a new commit, remove the call to Count from Count, and adding back in the implementation. Let me know if anything else is needed!

dl239 replied:
Well done.  There is compiling errors https://github.com/4paradigm/OpenMLDB/runs/6959539945?check_suite_focus=true#step:7:587
It also need to remove `ts_name` from here https://github.com/4paradigm/OpenMLDB/blob/40d3cbc98eb4a5750078f4849a8f097f6c89a1a1/src/cmd/openmldb.cc#L3587. thank you

TAldridgeAO committed:
Changed implementation of Count to deleted version, with no ts_name variable

TAldridgeAO committed:
update daily build
Merge https://github.com/0sirusD3m0n/OpenMLDB into 1974-remove-count-tsname-tablet-client

TAldridgeAO committed:
Merge branch '4paradigm:main' into 1974-remove-count-tsname-tablet-client

TAldridgeAO committed:
Removed ts_name from count call in openmldb.cc

TAldridgeAO committed:
Merge branch '1974-remove-count-tsname-tablet-client' of https://github.com/0sirusD3m0n/OpenMLDB into 1974-remove-count-tsname-tablet-client

vagetablechicken commented on src/client/tablet_client.cc:
```diff
@@ -776,12 +776,7 @@ bool TabletClient::Get(uint32_t tid, uint32_t pid, const std::string& pk, uint64
-bool TabletClient::Count(uint32_t tid, uint32_t pid, const std::string& pk, const std::string& idx_name,
-                         const std::string& ts_name, bool filter_expired_data, uint64_t& value, std::string& msg) {
-    ::openmldb::api::CountRequest request;
+     ::openmldb::api::CountRequest request;
```
intent, an extra space?

TAldridgeAO replied:
Not sure how that happened, guessing it was just an issue with VSCODE not liking my paste. I've updated it and pushed a new commit to fix.

TAldridgeAO committed:
removed extra space from tablet_client.cc
